config.js

```diff
@@ -768,6 +768,34 @@ config.NSFS_GLACIER_EXPIRY_TZ = 'LOCAL';
 // Format must be HH:MM:SS
 config.NSFS_GLACIER_EXPIRY_TIME_OF_DAY = '00:00:00';
 
+// NSFS_LOW_FREE_SPACE_CHECK_ENABLED if set to true will use the below mentioned
+// thresholds to determine if the writes should be denied even
+// before we hit ENOSPC more filesystem.
+config.NSFS_LOW_FREE_SPACE_CHECK_ENABLED = false;
+
+config.NSFS_STATFS_CACHE_SIZE = config.NSFS_DIR_CACHE_MAX_TOTAL_SIZE;
+config.NSFS_STATFS_CACHE_EXPIRY_MS = 1 * 1000;
```

Comment:
prefer to keep the NSFS_LOW_* configs next to eachother and not break them with this